packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -229,6 +229,10 @@ export type BaseInputProps = FormInputLabelProps &
      * true if popup is in expanded state
      */
     isPopupExpanded?: boolean;
+    /**
+     * sets the autocapitalize behavior for the input
+     */
+    autoCapitalize?: 'none' | 'sentences' | 'words' | 'characters';
```

Comment:
Is there a difference between setting "none" & undefined (not present) ?

Reply:
Yup, none will turn off autoCapitalize. `undefined` will reset it to its default behaviour. On mweb, when the type is email it will turn off autocapitalize but on native it will leave it on. passing it explicitly will turn  it off for all platforms to maintain parity